packages/pg/test/integration/client/sasl-scram-tests.js

```diff
@@ -45,12 +45,25 @@ if (!config.user || !config.password) {
   return
 }
 
-suite.testAsync('can connect using sasl/scram', async () => {
+suite.testAsync('can connect using sasl/scram (channel binding enabled)', async () => {
   const client = new pg.Client(config)
   let usingSasl = false
   client.connection.once('authenticationSASL', () => {
     usingSasl = true
   })
+  client.enableChannelBinding = true // default
```

Comment:
would suggest making the default false - adding new defaults in a patch version of `pg` is a risk for introducing breaking changes

Reply:
Fair point. In fact, even if you wanted to make it the default, it would probably make sense for it to get some testing in the wild by people who opt into it before doing that.